docs/UserGuide.md

```diff
@@ -16,6 +16,7 @@
     * [`author-config.csv`](#author-configcsv)
     * [`group-config.csv`](#group-configcsv)
 * [Analyzing Multiple Repos](#analyzing-multiple-repos)
+* [Using RepoSense with Netlify](#using-reposense-with-netlify)
```

Comment:
Perhaps `Quickstart/Tryout Reposense Using Netlify` instead to better capture the users' interest?

Reply:
Will go ahead with `Tryout Reposense Using Netlify`